IBPSA/Fluid/Movers/BaseClasses/IdealSource.mo

```diff
@@ -4,8 +4,11 @@ model IdealSource
   extends IBPSA.Fluid.Interfaces.PartialTwoPortTransport(show_T=false);
 
   // Quantity to control
-  parameter Boolean control_m_flow "= false to control dp instead of m_flow"
+  parameter Boolean control_m_flow "= false to m_flow"
     annotation(Evaluate = true);
+  parameter Boolean control_dp = not control_m_flow "= false to control dp"
```

Comment:
@marcusfuchs I improved the documentation. Can you please merge if all tests succeed.